packages/meditrak-server/src/apiV2/CRUDHandler/CRUDHandler.js

```diff
@@ -6,18 +6,18 @@
 import { singularise } from '@tupaia/utils';
 import { RouteHandler } from '../RouteHandler';
 import { resourceToRecordType } from '../../utilities';
-import { extractResourceFromEndpoint, extractChildResourceFromEndpoint } from './helpers';
+import { extractResourceFromPath, extractChildResourceFromPath } from './helpers';
 
 export class CRUDHandler extends RouteHandler {
   constructor(req, res) {
     super(req, res);
     const { recordId, parentRecordId } = this.params;
-    this.parentResource = parentRecordId ? extractResourceFromEndpoint(this.endpoint) : undefined;
+    this.parentResource = parentRecordId ? extractResourceFromPath(this.path) : undefined;
```

Comment:
Previously we were using a middleware to get the "endpoint" out of the path, essentially just stripping the `/v2` off the front of e.g. `/v2/surveys`. Now that we're using a router under `/v2`, express does this for us, so we can just use `req.path`